pkgs/development/web/playwright/driver.nix

```diff
@@ -196,28 +192,7 @@ let
     };
   });
 
-  browsers-mac = stdenv.mkDerivation {
```

Comment:
Love having this removed soon, but I think we should keep the impure builder available behind a flag for one release or so.

The reason is that we don't really have enough real-world testing for users on different darwin setups and we don't want to break working workflows if not needed. I don't think it's needed here and defaulting to your new approach while allowing opt-in to the old one would be perfect :)


Might as well call bin/playwright instead of cli.js

Reply:
> The reason is that we don't really have enough real-world testing for users on different darwin 

we use it on darwin aarch64 so I considered it covered and ready to fix problem reports asap, x86 doesn't work anyway because of https://github.com/NixOS/nixpkgs/issues/267894

Reply:
Thought about this and agree. I think removing this now is reasonable as users could still just call `playwright install` outside the sandbox, and we don't even set `PLAYWRIGHT_BROWSERS_PATH` yet.